integration/batching-with-context-esModuleInterop/batching.ts

```diff
@@ -770,3 +770,10 @@ function isObject(value: any): boolean {
 function isSet(value: any): boolean {
   return value !== null && value !== undefined;
 }
+
+function unwrap<T>(value: T | undefined | null): T {
```

Comment:
This makes sense, but we already use the term wrap/unwrap for dealing with wrapper types like `google.protobuf.StringValue`.

So can we use a name like `ensure`, `ensureSet`, `failIfUnset`, `check`...

Fwiw typically instead of a function, I'd write:

```
return ids.map(key => res.entites[key] ?? fail());
```

And define `fail` as:

```
export function fail(message?: string): never {
  throw new Error(message || "Failed");
}
```

Given that would most match what I'd write by hand, wdyt? Don't have a strong preference if you feel strongly about it being a function that takes a value.